### PR TITLE
CRS-1859 Update config to use new alerting channels for legacy

### PR DESCRIPTION
**Pull request by srowlands-moj**
* Update slack channels used for alerting and notification of deployments to the legacy specific ones

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -28,4 +28,4 @@ generic-service:
 # CloudPlatform AlertManager receiver to route prometheus alerts to slack
 # See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
 generic-prometheus-alerts:
```
Comment (ldlharper) [inline helm_deploy/values-dev.yaml:30]:
```suggestion
generic-prometheus-alerts:
  businessHoursOnly: true
```

File: helm_deploy/values-preprod.yaml [hunk 2]
```diff
@@ -31,4 +31,4 @@ generic-service:
 # CloudPlatform AlertManager receiver to route prometheus alerts to slack
 # See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
 generic-prometheus-alerts:
```
Comment (ldlharper) [inline helm_deploy/values-preprod.yaml:33]:
```suggestion
generic-prometheus-alerts:
  businessHoursOnly: true
```

**Review by ldlharper — COMMENTED** [review]
I've suggested a couple of changes.

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (srowlands-moj) [reply helm_deploy/values-dev.yaml:30]: Done 👍

File: helm_deploy/values-preprod.yaml (hunk 2 above)
Comment (srowlands-moj) [reply helm_deploy/values-preprod.yaml:33]: Done 👍

**Review by ldlharper — APPROVED** [review]
LGTM

**Pull request merged**